form-utils.js

```diff
@@ -93,6 +94,13 @@ const FormUtils = {
     )[0];
     return parseFloat(mode);
   },
+  canShowViaParams(doShow) {
+    if (["dev", "heroku"].includes(AppConstants.NODE_ENV) && doShow) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
Since you're just returning a boolean, you could one-liner this to:

```js
    return ["dev", "heroku"].includes(AppConstants.NODE_ENV) && doShow;
```

Reply:
👍🏽